curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-xmlhttprequest-method.english.md

```diff
@@ -161,7 +161,7 @@ tests:
   }
 </style>
 <h1>Cat Photo Finder</h1>
-<p class="message">
+<p class="message box">
```

Comment:
Here, you added the class to the solution, you should also add the class to the seed code.

Reply:
Oh, That`s my fault, Thank you for telling me